back/users/mail-mjml/signup_template-replace.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Chris": "{{if(options.name !== undefined)}}, {{ name }}{{/if}}",
+  "https://dailyidea-serverless-email-static-prod.s3.amazonaws.com": "{{ BUCKET_URL_PREFIX }}"
```

Comment:
why we use this hardcoded url here? it should be env vars of some kind because there dev and prod and we should also actually switch to cloudfront but that's will be done just be setting env var

Reply:
It replaces url with {{ BUCKET_URL_PREFIX }}, the hardcoded one is for editing mjml template so it will show all images correctly while editing it. So in actual html template will be {{ BUCKET_URL_PREFIX }} not a hardcoded one.

Reply:
Hm, but then it will end up with differences when u cant view what is deployed on dev only but not prod to test it

Reply:
@aldarund That code with hardcoded links is not used in the app, it's only to generate html templates, I added command `yarn compile-emails` it will compile emails from mjml and replace hardcoded urls like "https://dailyidea-serverless-email-static-prod.s3.amazonaws.com" to "{{ BUCKET_URL_PREFIX }}" so it actually uses env variable. This is just for convenience for editing mjml templates, otherwise it will not show images while editing mjml https://prnt.sc/ss2whf (I also replace Chris with the expression cause it breaks title to two lines, that will never happed with actual name).

Reply:
I understand what u are saying, my point is that when editing in mjml unless  we do deploy images into production that editing will be missing images and so editing wont be good. Especially considered that first its deployed to dev and only then to prod

Reply:
That urls are not important, if they stop working for some reasons they can be replaces with working ones even local, for net time edit. As they were already deployed on production I just used that urls. If they are removed from production and and then template needs to be edited again, then links needs to be updated. 

What do you offer here, reference all images locally?

Reply:
Im saying if for example we add new image - and we do it to dev initially. it wont be available and visible when editing in mjml.
I think something like different email templates for production vs dev than maybe if we cant parameterized it properly

Reply:
Then it can be s3 dev link instead. Not sure what do you offer as an alternative, two different mjml templates don't make sense it's much more confusing, it's simpler to replace links in one in case image not accessible anymore. There is no way to do any params in mjml, it's very simple tool. And I don't see any need to make anything here as this is just for editing. Maybe to use local images, but not sure if mjml preview plugin will load local images, also it will not allow to share template in online editor https://mjml.io/try-it-live which is very convenient for reviews. Actual html email template has BUCKET_URL_PREFIX env variable, so it will work on all environments.

Reply:
As i said the problem is that we add image to dev, we want to preview how template will look with it -> but its not on production and hence the preview will fail and not any useful for dev. Isnt it a problem?

Reply:
But what solution do you offer here?